### ISSUE TITLE
No distinction between "Explicit many-to-many relations" and "Relation tables" provided in documenation

### ISSUE THREAD

**Issue by Flip-Liquid** [prisma/prisma2-docs#4114]
On https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations

Confused as to whether these are actually different categories and I'm missing something, or if there is a distinction being drawn without any difference. 

Here's the prisma model for the "Explicit many-to-many relations"
```
model Post {
  id         Int                 @id @default(autoincrement())
  title      String
  categories CategoriesOnPosts[]
}

model Category {
  id    Int                 @id @default(autoincrement())
  name  String
  posts CategoriesOnPosts[]
}

model CategoriesOnPosts {
  post       Post     @relation(fields: [postId], references: [id])
  postId     Int // relation scalar field (used in the `@relation` attribute above)
  category   Category @relation(fields: [categoryId], references: [id])
  categoryId Int // relation scalar field (used in the `@relation` attribute above)
  assignedAt DateTime @default(now())
  assignedBy String

  @@id([postId, categoryId])
}
```

and here it is for the "Relation tables":
```
model Post {
  id         Int                 @id @default(autoincrement())
  title      String
  categories CategoriesOnPosts[]
}

model Category {
  id    Int                 @id @default(autoincrement())
  name  String
  posts CategoriesOnPosts[]
}

model CategoriesOnPosts {
  post       Post     @relation(fields: [postId], references: [id])
  postId     Int
  category   Category @relation(fields: [categoryId], references: [id])
  categoryId Int
  assignedAt DateTime @default(now())
  assignedBy String

  @@id([postId, categoryId])
}
```

These are identical save for the comments.

**Comment by janpio** [comment]
A relation table is indeed needed to create an explicit many-to-many relation, so they are closely related and tightly coupled. It is weird that they are separated in the docs.

**Comment by nikolasburk** [comment]
Agree with @janpio, the structure makes this a bit confusing! I don't think "Relation tables" need an extra section, instead we should make sure that the terminology is properly explain in the "Explicit many-to-many relations" section.

**Comment by jharrell** [comment]
Resolved by #4792
